.circleci/config.yml

```diff
@@ -46,24 +46,24 @@ jobs:
           # Limiting the workers of Jest to 10
           # as the build otherwise dies due to resouce restrictions.
           command: yarn lint --maxWorkers=10
-  test_unit_node_6:
-    <<: *node6Environment
+  test_unit_node_8:
+    <<: *node8Environment
     steps:
       - checkout
       - restore-cache: *restore_yarn_cache
       - run: *yarn_install
       - save_cache: *save_yarn_cache
       - run: *unit_test
-  test_unit_node_8:
-    <<: *node8Environment
+  test_unit_node_10:
```

Comment:
You can add node_11 if you want.

Reply:
instead or in addition to 10?

Reply:
Interesting idea, what is the thought behind it? My first reaction to it was to just wait for Node 12 instead since 11 will never be lts.